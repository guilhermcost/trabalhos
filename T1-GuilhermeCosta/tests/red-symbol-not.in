// testa !  

if !(a) 
return 0;

// fim 

