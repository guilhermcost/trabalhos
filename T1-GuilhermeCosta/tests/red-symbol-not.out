(3,KEY,"if")
(3,ERROR,"!")
(3,SYM,"(")
(3,ID,"a")
(3,SYM,")")
(4,KEY,"return")
(4,NUM,"0")
(4,SYM,";")
